packages/oncokb-frontend-commons/src/components/OncoKB.tsx

```diff
@@ -0,0 +1,270 @@
+import { MobxCache, OncoKbCardDataType } from 'cbioportal-utils';
```

Comment:
What's the difference now between MobxCache and in library Cache?

Reply:
I think `MobxCache` is an unused import now. We should try cleaning up all the other unused imports as well.